Update head to include defaults from docs-italia

pdavide opened the pull request:
- add social tags
- add noindex
- remove unused icons

francescozaia reviewed (approved):
La soluzione è interessante e formalmente ok, ma ho 2 domande:
- non sono convinto dell'utilizzo di asset che dipendono da un altro progetto... peraltro quei riferimenti perdono di significato se usati su RTD o in locale?
- il tema ha conoscenza di `{{ MEDIA_URL }}`?

pdavide commented:
@francescozaia 
- nel tema è impostata la variabile `{{ MEDIA_URL }}`
- sistemo la PR in modo da mantenere la compatibilità con RTD e con l'ambiente locale

pdavide committed:
Update head to include defaults from docs-italia

- add noindex
- remove unused icons
- exclude docs-italia head tags for RTD and local development

francescozaia commented:
Per me si può mergiare, grazie
